lessons/main.swift

```diff
@@ -7,4 +7,92 @@
 
 import Foundation
 
+func readDoubleParameter(parameterName: String? = nil, isPositive: Bool = false) -> Double {
+    repeat {
+        if let parameterName = parameterName {
+            print("Введите \(parameterName): ")
+        }
+        let stringValidator = readLine()
+        if let doubleValidator = Double(stringValidator ?? "") {
+            if isPositive && doubleValidator < 0 {
+                continue
+            }
+            return Double(doubleValidator)
+        } else {
+            print("Вы ввели не число")
+        }
+    } while (true)
+}
 
+func OneTask() {
+    print("Решение квадратного уравнения!")
+
+    print("Введите коэффициенты для квадратного уравнения (ax^2 + bx + c = 0):")
+    let a = readDoubleParameter(parameterName: "a")
+    let b = readDoubleParameter(parameterName: "b")
+    let c = readDoubleParameter(parameterName: "c")
+
+    let discr = b * b - 4 * a * c;
+
+    if discr > 0 {
+        let x1 = (-b + sqrt(discr)) / (2 * a)
+        let x2 = (-b - sqrt(discr)) / (2 * a)
+        print(String(format: "x1 = %.2f; x2 = %.2f", x1, x2))
+    } else if discr == 0 {
+        let x1 = (-b) / (2 * a)
+        print(String(format: "x1 = %.2f", x1))
+    } else {
+        print("Корней нет!")
+    }
+}
+
+//OneTask()
+func TwoTask() {
+    print("Даны катеты прямоугольного треугольника. Найти площадь, периметр и гипотенузу треугольника.")
+
+    let a = readDoubleParameter(parameterName: "первый катет", isPositive: true)
+    let b = readDoubleParameter(parameterName: "второй катет", isPositive: true)
+
+    let c = sqrt(a * a + b * b)
+    let P = a + b + c
+    let S = (a * b) / 2
+    print(String(format: "Гипотенуза = %.2f; Периметр = %.2f; Площадь = %.2f", c, P, S))
+}
+
+//TwoTask()
+
+func ThreeTask() {
```

Comment:
```suggestion
func threeTask() {
```
в каком языке так пишут?